app/models/pet.rb

```diff
@@ -1,6 +1,7 @@
 class Pet < ApplicationRecord
   belongs_to :user
   has_many :rents, dependent: :destroy
+  has_many :reviews, dependent: :destroy
```

Comment:
do we want to remove all reviews with the deletion of the user?

Reply:
It depends on deletion of a pet.